src/pages/UserSettingsPage/UserSettingsPage.tsx

```diff
@@ -0,0 +1,285 @@
+            </Select>
+          </FormControl>
+          <Controller
+            control={control}
+            name="user_pic"
+            render={({
+              field: { ref, onChange, onBlur, value, name },
+              fieldState: { isTouched, isDirty, error },
+              formState,
+            }) => (
+              <TextField
+                label="User Picture"
+                variant="outlined"
+                sx={{ mb: 2, width: '100%' }}
+                inputRef={ref}
+                helperText={
+                  !!error && isTouched && <>URL to picture. May be blank.</>
+                }
+                error={!!error && isTouched}
+                onChange={onChange}
+                onBlur={onBlur}
+                placeholder="https://example.com/picture.jpg"
+                value={value}
+                name={name}
+              />
+            )}
+            rules={{
+              required: false,
+              pattern: /^https?:\/\/.*\.(?:png|jpg|jpeg|gif|svg|webp)$/
+            }}
+          />
+          <Controller
+            control={control}
+            name="user_bio"
+            render={({
+              field: { ref, onChange, onBlur, value, name },
+              fieldState: { isTouched, isDirty, error },
+              formState,
+            }) => (
+              <TextField
+                label="User Bio"
+                variant="outlined"
+                multiline
+                minRows={2}
+                sx={{ mb: 2, width: '100%' }}
+                inputRef={ref}
+                helperText={
+                  !!error && isTouched && <>Please say something about yourself.</>
+                }
+                error={!!error && isTouched}
+                onChange={onChange}
+                onBlur={onBlur}
+                value={value}
+                name={name}
+              />
+            )}
+            rules={{ required: true }}
+          />
+          <Controller
+            control={control}
+            name="zipcode"
+            render={({
+              field: { ref, onChange, onBlur, value, name },
+              fieldState: { isTouched, isDirty, error },
+              formState,
+            }) => (
+              <TextField
+                label="Zip Code"
+                variant="outlined"
+                sx={{ mb: 2, width: '100%' }}
+                inputRef={ref}
+                placeholder={"20036"}
+                helperText={
+                  !!error && isTouched && <>Please enter a valid zip code.</>
+                }
+                error={!!error && isTouched}
+                onChange={onChange}
+                onBlur={onBlur}
+                value={value}
+                name={name}
+              />
+            )}
+            rules={{
+              required: true, 
+              pattern: /^\d{5}(?:[-\s]\d{4})?$/
+            }}
+          />
+          <FormControl fullWidth sx={{ mb: 2 }}>
```

Comment:
Same question here for the other FormControl input -- does this value show up in the console.log?

Reply:
Same answer. I'll check to see if the values are updated.